lib/spotlight_search/exportable_columns.rb

```diff
@@ -0,0 +1,71 @@
+module SpotlightSearch
+  module ExportableColumns
+    mattr_accessor :enabled_columns, :export_enabled
+
+    module ClassMethods
+      # Enables or disables export and specifies which all columns can be
+      # exported. For enabling export for all columns in all models
+      # 
+      #   class ApplicationRecord < ActiveRecord::Base
+      #     enable_export_for_columns enabled: true
+      #   end
+      #
+      # For disabling export for only specific models
+      #
+      #   class Person < ActiveRecord::Base
+      #     enable_export_for_columns enabled: false
+      #   end
+      #
+      # For allowing export for only specific columns in a model  
+      #
+      #   class Person < ActiveRecord::Base
+      #     enable_export_for_columns enabled: true, only: [:created_at, :updated_at]
```

Comment:
@sajinmp - does it also support exporting methods?

Reply:
@sajinmp - Also, I was wondering if we could have the following naming convention --

```
class Person < ActiveRecord::Base
  exportable, only: [:created_at, :updated_at]
```

If they add exportable, it's pretty obvious they want it enabled. What do you think?

Reply:
@sajinmp - any update on the feasibility of this bro? We can then proceed to merge this PR :)

Reply:
@sajinmp - any update on this bro? I hope you are receiving the PR notifications :D

Reply:
@mikevic Haven't taken up this for some time. Not receiving the notification for comments in this. Will check why

exportable was used somewhere else if i'm not wrong and didn't want all the names to be same. 

enabled/disabled are for disabling for a particular model, if its enabled in the ApplicationRecord globally.

The pending thing is to build the pop up for the selection of columns in the view. I'll see if i can change the name to exportable, if its not used anywhere else.

Reply:
@sajinmp -sure bro. Let me know :) I hope you are getting this notification!

Reply:
@mikevic Got it. I had only changed the settings in aiesec. Not in commutatus. :D

Reply:
Have changed the name to `export_columns`